### PR TITLE
MC-13026 addition of api docs for images

### PR DESCRIPTION
**Pull request by razine-bensari**
#### Changes made

- addition of images from stackpath.

#### Related PRs
- [list-images](https://github.com/cloudops/cloudmc-stackpath-plugin/pull/19)

### CODE REVIEW

File: source/includes/stackpath/_images.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+## Images
+
+StackPath Edge Computing makes use of images to deploy virtual machines within a workload. The image can be from StackPath default images or
+a custom one. Only non-deprecated images are returned by default.
+
+Deploy and manage your Images.
```
Comment (patrickspensieri) [inline source/includes/stackpath/_images.md]: should be "your images" in lowercase

File: source/includes/stackpath/_images.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/images</code>
+
+Retrieve a list of all images in a given [environment](#administration-environments).
+
+Attributes | &nbsp;
+------- | -----------
+`id`<br/>*string* | An image's unique identifier. It is the combination of /.
```
Comment (patrickspensieri) [inline source/includes/stackpath/_images.md]: Looks like we're missing something here 👀 (also in attribute table for get image)

File: source/includes/stackpath/_images.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+`reference`<br/>*string* | The reference of the image.
+
+<!-------------------- RETRIEVE AN IMAGE -------------------->
+
+### Retrieve an image
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/stackpath/test-area/images/image?family=ubuntu&tag=v20201110"
```
Comment (patrickspensieri) [inline source/includes/stackpath/_images.md]: Should the `id` be `c6f9f9c4-31a3-4114-946e-0bb82883301a` here?

Comment (y0geshdev) [reply source/includes/stackpath/_images.md]: @razine-bensari Based on your other recent PR, the image id should be a combination of `stackId/family/tag` ... no?

File: source/includes/stackpath/_images.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/images/:id</code>
+
+Retrieve an image in a given [environment](#administration-environments).
+
+Attributes | &nbsp;
+------- | -----------
+`id`<br/>*string* | An image's unique identifier. It is the combination of /.
```
Comment (y0geshdev) [inline source/includes/stackpath/_images.md]: Same as Pat's above comment. Should we mention that the id is made up by joining stackId, family name and tag?

**Review by y0geshdev — CHANGES_REQUESTED** [review]
Some minor suggestions.

File: source/includes/stackpath/_images.md (hunk 4 above)
Comment (razine-bensari) [reply source/includes/stackpath/_images.md]: https://github.com/cloudops/cloudmc-api-docs/pull/268/commits/bc7606f72cc3b43fea9d39f4c1b787a147714c17

File: source/includes/stackpath/_images.md (hunk 3 above)
Comment (razine-bensari) [reply source/includes/stackpath/_images.md]: https://github.com/cloudops/cloudmc-api-docs/pull/268/commits/bc7606f72cc3b43fea9d39f4c1b787a147714c17

File: source/includes/stackpath/_images.md (hunk 1 above)
Comment (razine-bensari) [reply source/includes/stackpath/_images.md]: https://github.com/cloudops/cloudmc-api-docs/pull/268/commits/bc7606f72cc3b43fea9d39f4c1b787a147714c17

**Review by y0geshdev — APPROVED** [review]
Thank you for the changes. 🙏 
Good work.

File: source/includes/stackpath/_images.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+`reference`<br/>*string* | The reference of the image.
+
+<!-------------------- RETRIEVE AN IMAGE -------------------->
+
+### Retrieve an image
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/stackpath/test-area/images/:id"
```
Comment (patrickspensieri) [inline source/includes/stackpath/_images.md]: Very minor, but this should use an example id. In this case `https://cloudmc_endpoint/v1/services/stackpath/test-area/images/a8050b2b-39eb-4929-bce5-1af42055903e/ubuntu/v20201110`

**Review by patrickspensieri — COMMENTED** [review]
Minor detail, just to use an example id in the example request to be consistent with the rest of our docs:)

File: source/includes/stackpath/_images.md (hunk 5 above)
Comment (razine-bensari) [reply source/includes/stackpath/_images.md]: Good catch!

Comment (razine-bensari) [reply source/includes/stackpath/_images.md]: https://github.com/cloudops/cloudmc-api-docs/pull/268/commits/8069c826dcccc3d18f3f7f28d871ae8a2ab5a57a

**Review by patrickspensieri — APPROVED** [review]
✔️

**Pull request merged**